tests/test_settings.cpp

```diff
@@ -89,20 +91,23 @@ TEST_F(TestSettings, keysReturnsKeysFromSingleHandler)
 
 TEST_F(TestSettings, keysReturnsKeysFromMultipleHandlers)
 {
-    auto some_keychains =
-        std::array{std::set({QStringLiteral("asdf.fdsa"), QStringLiteral("blah.bleh")}),
-                   std::set({QStringLiteral("qwerty.ytrewq"), QStringLiteral("foo"), QStringLiteral("bar")}),
-                   std::set({QStringLiteral("a.b.c.d")})};
+    auto some_keychains = std::array{
+        std::set({QStringLiteral("asdf.fdsa"), QStringLiteral("blah.bleh")}),
+        std::set({QStringLiteral("qwerty.ytrewq"), QStringLiteral("foo"), QStringLiteral("bar")}),
+        std::set({QStringLiteral("a.b.c.d")})};
 
     for (auto i = 0u; i < some_keychains.size(); ++i)
     {
         auto mock_handler = std::make_unique<MockSettingsHandler>();
-        EXPECT_CALL(*mock_handler, keys).WillOnce(Return(some_keychains[i])); // copies, so ok to modify below
+        EXPECT_CALL(*mock_handler, keys)
+            .WillOnce(Return(some_keychains[i])); // copies, so ok to modify below
         MP_SETTINGS.register_handler(std::move(mock_handler));
     }
 
-    auto all_keys = std::reduce(std::begin(some_keychains), std::end(some_keychains), // hands-off clang-format
-                                std::set<QString>{}, [](auto&& a, auto&& b) {
+    auto all_keys = std::reduce(std::begin(some_keychains),
+                                std::end(some_keychains), // hands-off clang-format
```

Comment:
what does this comment mean?

Reply:
It was an artificial way to prevent clang-format from keeping some parameters in a single line, but others in their own lines. Doesn't appear to be necessary any longer.